### ISSUE TITLE
Mark images with missing image descriptions/alt text

### ISSUE THREAD

**Issue by nevillepark** [tuskyapp/Tusky#3084]
Some instances have implemented [custom CSS that highlights media attachments missing descriptions/alt text](https://gist.github.com/FiXato/3de505b04efefd49a1e3568b70545234) with a contrasting border. It easily allows you to see if an image description is missing. A similar feature would be especially useful in Tusky, as you wouldn't even have to open or long press the media to check. It would be helpful for those of us who don't boost posts with undescribed media, or who would like to add descriptions to posts that don't have them.

I've seen the issues asking for alt text *reminders*, but this would be different as it applies to other users' already published posts.

* * * *
- Tusky Version: 20.0
- Android Version: 11
- Android Device: Moto G5 Plus
- Mastodon instance (if applicable): flipping.rocks

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by Tak** [comment]
This was implemented in #2942 for Tusky 21

**Comment by nevillepark** [comment]
Thank you so much!
